javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <script src="spirit_nuevo_2.js"></script>
```

Comment:
Its recommended to add script tag at the end of body tag.

the script file name added here and script file committed doesn't match. verify that.